source/docs/advanced/theme/translations.md

```diff
@@ -0,0 +1,93 @@
+---
+id: translations
+title: Translate your application
+---
+
+Big e-commerce websites often need to translate their content in several languages to make sure that they reach a broader range of customers.
+
+In this documentation, we will see how this works for static content in your application.
+
+<blockquote class="note">
+If you want to configure your application to support multiple languages, please refer to [Configure multiple stores](/docs/advanced/production-ready/multistore.html).
```

Comment:
```suggestion
Often you may want to change dynamic content (CMS pages, product descriptions…) in addition to translating the UI. If you want to configure your application to support multiple languages, please refer to [Configure multiple stores](/docs/advanced/production-ready/multistore.html).
```

Reply:
I saw it was also mentioned in the last section, so feel free to discard this suggestion if you’d like.